editor/components/icon-button/style.scss

```diff
@@ -14,4 +14,8 @@
 			color: $blue-medium;
 		}
 	}
+
+	span {
+		font-size: 12px;
```

Comment:
This styling seems very targeted at the heading use-case. Is `IconButton` with small adjacent text a common enough pattern it should be engrained into the `IconButton` component itself?